tinker/faculty_bios/__init__.py

```diff
@@ -58,27 +58,36 @@ def index_cache(self, username):
         else:
             show_courseleaf_button = False
 
-        # the faculty special admins should be able to see every bio, based on school.
+        show_special_admin_view = False
         if 'Tinker Faculty Bios - Admin' in session['groups'] or 'Administrators' in session['groups']:
             show_special_admin_view = True
-            show_create = True
-
-            # This nastiness is to maintain order and have the class value
-            all_schools = [
-                {'cas': 'College of Arts and Sciences'},
-                {'caps': 'College of Adult and Professional Studies'},
-                {'gs': 'Graduate School'},
-                {'sem': 'Bethel Seminary'},
-                {'bu': 'Administration with Faculty Status'},
-                {'other-category': 'Other'}
-            ]
-        else:  # normal view
-            show_special_admin_view = False
-            show_create = len(forms) == 0 \
-                          or 'Tinker Faculty Bios - CAS' in session['groups'] \
-                          or 'Tinker Faculty Bios - CAPS and GS' in session['groups'] \
-                          or 'Tinker Faculty Bios - SEM' in session['groups'] \
-                          or self.base.is_user_in_web_author_groups()
+
+        # This nastiness is to maintain order and have the class value
+        all_schools = [
+            {'cas': 'College of Arts and Sciences'},
+            {'caps': 'College of Adult and Professional Studies'},
+            {'gs': 'Graduate School'},
+            {'sem': 'Bethel Seminary'},
+            {'bu': 'Administration with Faculty Status'},
+            {'other-category': 'Other'}
+        ]
+
+        class_list = []
+        if 'Tinker Faculty Bios - CAS' in session['groups']:
```

Comment:
might be worth mentioning the new functionality here?

Reply:
I added a comment for this